Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -23,6 +23,8 @@
     public final XCANTalon rightDrive;
     public final XCANTalon rightDriveFollower;
     
+    public boolean precisionMode;
```

Comment:
might be nice if this was a booleanproperty so we could see what mode we were in on the dashboard?